bank_management/operations/bankCounter.go

```diff
@@ -1,27 +1,67 @@
 package operations
 
-import "time"
+import (
+	"fmt"
+	"time"
+)
 
-type bankCounterInterface interface {
+type bankInterface interface {
```

Comment:
do not use go Keywords in naming , this interface can be more appropriately names for ex, bankOperation or bankServices